pom.xml

```diff
@@ -260,6 +260,11 @@
       <version>1.10.8</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.reflections</groupId>
+      <artifactId>reflections</artifactId>
+      <version>0.9.12</version>
```

Comment:
good part of it is that all dependencies of this library are `optional` 
https://github.com/ronmamo/reflections/blob/master/pom.xml#L60
https://maven.apache.org/guides/introduction/introduction-to-optional-and-excludes-dependencies.html#how-do-optional-dependencies-work
so we will not gain all transitive dependencies to our uber jar `-all.jar`.

@strkkk , please prove that Guava implementation will be enough for us.
so we might skip this dependency completely.

Reply:
@gaurabdg , please downgrade to `0.9.11` it will resolve issue wercker.
We can upgrade to latest version of library in separate PR.

Reply:
I pushed extra commit to your branch to fix problem with xwiki.
please feel free to squash it to your commit at any time.

Reply:
Done